sklearn/linear_model/passive_aggressive.py

```diff
@@ -36,6 +36,27 @@ class PassiveAggressiveClassifier(BaseSGDClassifier):
 
         .. versionadded:: 0.19
 
+    early_stopping : bool, default=False
+        Whether to use early stopping to terminate training when validation.
+        score is not improving. If set to True, it will automatically set aside
+        a fraction of training data as validation and terminate training when
+        validation score is not improving by at least tol for
+        n_iter_no_change consecutive epochs.
+
+        .. versionadded:: 0.20
+
+    n_iter_no_change : int, default=1
+        Number of iterations with no improvement to wait before early stopping.
```

Comment:
For googlability, we should mention that this parameter is also named "patience" in other libraries.

Reply:
We should also recommend to set it to a large enough value such as 5 or 10 to avoid premature stopping.